Gems/ROS2/Code/Include/ROS2/Georeference/GeoreferenceBus.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Math/Vector3.h>
+
+namespace ROS2
+{
+    //! Interface that allows to convert between level and WSG84 coordinates.
+    class GeoreferenceRequests
+    {
+    public:
+        using WGS84Coordinate = AZStd::array<double,3>;
```

Comment:
Missing array include header. It might be in other headers, but you should not relay on it (include directly).
[Include what you use](https://github.com/include-what-you-use/include-what-you-use#instructions-for-users ).

Reply:
Another point is that array type is unnamed - people can get confused about whether latitude / longitude or altitude comes first.

How about a struct with a constructor from Vector3 and named fields (latitude, longitude, altitude?).

The second aspect is naming. In case of anonymous fields it would be correct to name it Vector3double or similar as it is generic. But consider the struct and named fields.